Task1_Sum_of_List.py

```diff
@@ -0,0 +1,17 @@
+from pip._vendor.msgpack.fallback import xrange
+
+
+def generate_random():
+    import random
+    some_list = random.sample(xrange(100), 10)
+    sum_of_list = sum(some_list)
+    average = sum_of_list / len(some_list)
+    print(some_list, sum_of_list, average)
```

Comment:
this is not the purpose of this function to print something, please move it out of the function